lib/membrane_ogg/payloader/opus.ex

```diff
@@ -95,6 +104,12 @@ defmodule Membrane.Ogg.Payloader.Opus do
 
   @impl true
   def handle_caps(:input, caps, _ctx, state) do
+    if caps.channels > 2,
+      do:
+        IO.warn(
+          "Can't properly encode opus stream with #{caps.channels} channels. Resultant file will be unreadable"
```

Comment:
```suggestion
          "Can't properly encode opus stream with #{caps.channels} channels. Output will not be a valid Opus stream"
```
First of all... encoding? I thought that this is the Opus Payloader. Secondly, I'm not even fully positive that this is true: https://datatracker.ietf.org/doc/html/rfc7845#section-5.1.1
This comment suggests that Opus with more than 2 channels is illegal. Unless it is, I would try to rephrase it so that it indicates lack of support